### ISSUE TITLE
Suggestion: Place value boxes outside of cards

### ISSUE THREAD

**Issue by AlbertRapp** [rstudio/shinyuieditor#220]
Hi,

I noticed that value boxes always get wrapped into cards (at least I couldn't find out how to change this default behavior in the UI editor). I think the resulting app would look better, if value boxes don't have an additional border around them.

Currently it looks like this:

![image](https://github.com/rstudio/shinyuieditor/assets/65388595/d84f327e-505a-4191-a7b0-5fe711b9b1ed)

Here's what I think would be nicer default behavior (from `bslib::bslib_theme_preview()`):

![image](https://github.com/rstudio/shinyuieditor/assets/65388595/cb23b752-d1ae-49a9-bc66-bbf7d4f43405)

Best
Albert

**Comment by nstrayer** [comment]
Agreed. Currently, this is because only items that are gridlayout aware can go directly on the grid. That being said, there is a wrapper in gridlayout already that adds this functionality so it should be as simple as writing out `grid_place(area="AREA", value_box(...))`.

**Comment by nstrayer** [comment]
Another option will be once the api for the column layout stuff in `bslib` is stable, that will be added to the editor, so no gridlayout will be needed. Once the next release of the editor is out I can look at getting this figured out.

**Comment by AlbertRapp** [comment]
This change worked nicely for me. Thanks. One thing I noticed is that when I reopen the UI editor the value boxes are displayed as unknown UI at the **bottom** of the card. But in the app they are at the top of the card (above the chart). Is that intended behavior? 

![image](https://github.com/rstudio/shinyuieditor/assets/65388595/dd0bd9df-3106-4b3b-b85d-1cc3054ef448)
